asm-to-pil/src/romgen.rs

```diff
@@ -214,11 +215,24 @@ pub fn generate_machine_rom<T: FieldElement>(
             parse_function_statement("_loop;"),
         ])]);
 
+        let latch = instruction_flag(RETURN_NAME);
+        let last_step = "_block_enforcer_last_step";
+        let operation_id_no_change = "_operation_id_no_change";
```

Comment:
could we also make these two user-generated?

Reply:
Which two of the 3? ^^

We currently error if the used sets the latch or operation ID if the machine has a PC (i.e., is a VM). I think that's fine, a VM is a higher-level concept, so there's some magic around that (like the magic `return` instruction you never have to declare). Anyway, I think changing that is out of scope for this PR.